src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1033,6 +1138,23 @@ public static Collection<FunctionMetadata> ReadFunctionMetadata(ScriptHostConfig
             // determine the script type based on the primary script file extension
             functionMetadata.ScriptType = ParseScriptType(functionMetadata.ScriptFile);
 
+            JToken isDirect;
+            if (functionConfig.TryGetValue("configurationSource", StringComparison.OrdinalIgnoreCase, out isDirect))
+            {
+                var value = isDirect.ToString().ToLowerInvariant();
+                if (value == "attributes")
+                {
+                    functionMetadata.IsDirect = true;
+                }
+                else if (value == "config")
+                {
+                }
+                else
```

Comment:
It's not empty. It's leaving the flag set to false without throwing.  :) 
There's 3 cases here: true, false, throw. 

---
In reply to: [129927238](https://github.com/Azure/azure-webjobs-sdk-script/pull/1717#discussion_r129927238) [](ancestors = 129927238)